woocommerce/woocommerce
# Replace i18n textdomain 'woocommerce-admin' with 'woocommerce'

### Description

This PR fixes #32175 by changing the text-domain strings across:

- JS/TS Files in:
  - ./packages/js/
  - ./plugins/woocommerce-admin/client/

- PHP Files in:
  - ./plugins/woocommerce/src/Admin/
  - ./plugins/woocommerce/src/Internal/
  - ./plugins/woocommerce/includes/react-admin/

- Markdown files for related documentation and examples in the above folders

#### JS
The JS/TS files were modified by using jscodeshift, with the transformation codified in https://github.com/woocommerce/woocommerce/blob/6f23a5ae68323f4457d644502aabe87391a4c9eb/tools/monorepo-merge/codemods/rename-intl-domain.js

These commands were ran from the monorepo-merge folder:

```sh
pnpx jscodeshift ../../packages/js/ -t codemods/rename-intl-domain.js --extensions=js,jsx,ts,tsx
pnpm run lint -- --fix
```
```sh
pnpx jscodeshift ../../plugins/woocommerce-admin/client -t codemods/rename-intl-domain.js --extensions=js,jsx,ts,tsx
pnpm run --filter "@woocommerce/admin-library" lint:js -- --fix
```

The lint-fix was necessary as there were some lint corrections applied by jscodeshift which violated our lint rules - it doesn't localise to the repo's lint rules.


#### PHP
The PHP files were modified using the existing script for wp-textdomain https://github.com/woocommerce/woocommerce/blob/6e8d620f00109b9c37083c778d30e6357d32a05f/plugins/woocommerce-admin/bin/package-update-textdomain.js

The existing script was copied over to woocommerce-admin and modified to take multiple folders. As I understand it, the original copy of this script was used to correct the text-domain during the build process for woocommerce-core.

This command was ran from the root of the repository:

```sh
pnpm run --filter "@woocommerce/admin-library" packages:fix:textdomain ../woocommerce/src/Admin/ ../woocommerce/src/Internal/ ../woocommerce/includes/react-admin/
```

#### Markdown
The .md files were modified by hand, by doing a find-in-folder of "woocommerce-admin" and inspecting if they were relevant. (There was the option to use jscodeshift + remark to do this too but remark ended up including a whole bunch of lint formatting that it thought it knew best 😠 ) 



### How to test the changes in this Pull Request:

1. Do the usual repo setup, i.e install and build 
2. Check that tests all still run
3. Check that i18n works correctly by changing the language in a running copy


### Changelog entry

i18n text domain strings were changed from 'woocommerce-admin' to 'woocommerce' in the files that were moved over from woocommerce-admin repo


### FOR PR REVIEWER ONLY:
